pkg/pillar/hypervisor/xen.go

```diff
@@ -195,6 +195,9 @@ func (ctx xenContext) CreateDomConfig(domainName string, config types.DomainConf
 		if ds.Format == zconfig.Format_CONTAINER {
 			p9Strings = append(p9Strings,
 				fmt.Sprintf("'tag=share_dir,security_model=none,path=%s'", ds.FileLocation))
+		} else if ds.IsBlank {
+			p9Strings = append(p9Strings,
+				fmt.Sprintf("'tag=%s,security_model=none,path=%s'", ds.MountDir, ds.FileLocation))
```

Comment:
I don't see it consumed anywhere. This appends it to the xen config file, so it will get passed to the VM, but we need to modify the VM init script to mount them.

Reply:
Also, we need parallel code for the other hypervisors, primarily kvm and containerd.

Reply:
>I don't see it consumed anywhere. This appends it to the xen config file, so it will get passed to the VM, but we need to modify the VM init script to mount them.

Yes, it is pending thing in PR. I am working on it.

Reply:
>Also, we need parallel code for the other hypervisors, primarily kvm and containerd.

Thanks for pointing it out. I will add it.